config/templates/wp-config.php.erb

```diff
@@ -15,7 +15,7 @@
 
```

Comment:
Is this template used? I'm not clear of difference between this and `dist` version below.

Reply:
Yes, used during Capistrano deployment (`deploy.rb`). Differences: Composer autoload path, pulls DB info from `config/database.yml`